control

```diff
@@ -582,6 +595,26 @@ class StepInit(Context, collections.Callable):
         self.step_timeout = 0
         # This is incremented by steps, reset for execution
         self.index = 0
+        # Setup for job data collection, clear all defaults.
+        for item in [job.sysinfo.after_iteration_loggables,
+                     job.sysinfo.before_iteration_loggables,
+                     job.sysinfo.boot_loggables,
+                     job.sysinfo.test_loggables]:
+            item.clear()
+        for _type, items in SYSINFOS.items():
+            # Avoid calling private job._add_sysinfo_loggable()
+            if _type == 'job':
+                on_every_test = False
+            else:
+                on_every_test = True
+            for dargs in items.get('commands', []):
+                # Easier than dealing with parameter conflicts
+                dargs.update(dict(on_every_test=on_every_test))
+                job.add_sysinfo_commnad(**dargs)
```

Comment:
This looks like it's misspelled (command), but in my testing I don't see any errors logged. (I also don't see the files getting created)